src/StatusQ/Components/StatusChatListAndCategories.qml

```diff
@@ -136,16 +133,15 @@ Item {
                         menuButton.tooltip: statusChatListAndCategories.categoryMenuButtonToolTip
 
                         originalOrder: model.position
-                        categoryId: model.categoryId
+                        categoryId: model.itemId
```

Comment:
I see you use `itemId` here as well. Maybe then something else needs to be changed so it's not as confusing?

Reply:
@jrainville I explained above what's like that. The other way is to have totally different component in the `statusq` which will know how to display chats/channels/category channels based on the single model from the backend. Actually backend model is very simple, we can look into it as a model whose items are either final items or group of items(in case of channel's category). 

It was a bit harder to apply classic `composite design pattern` in Nim, although it reminds of it, but once we switch to c++ it will be it (a common and easy technique for such cases).